returnn/search.py

```diff
@@ -137,17 +135,18 @@ def create_returnn_config(
         """
         assert device in ["gpu", "cpu"]
         original_config = returnn_config.config
+        res = copy.deepcopy(returnn_config)
         assert "network" in original_config
```

Comment:
is this assert still valid nowadays? e.g. in case of staged network dict or network definition in prolog/epilog.